BestFloww/BestFlow
# Integrate IntentAnalyzer

## Pull Request Summary
Integrate intent analyzer into the transcript interactor. Also, split the current interactor into two interactors for CA. And fixed some typos regarding total_children attribute and arrow functions in the intent model.

### Change types


### Testing
yes

### How to manually test this PR (QA instructions)
I am using Postman to send get and post requests and don't know if there's another way to send a get request from our FE rn. Anyway, if you have Postman, upload a transcript to the DB and send a GET request. Notice that the intents are returned in a list exactly as according to intent analyzer, and it is different from how it appears in the database prior to analysis. If there is nothing in the db, an empty list is returned.

## Questions
